epub34/overview/index.html

```diff
@@ -777,11 +755,10 @@ <h3>EPUB 3.3: 2022</h3>
 			<section id="epub34" class="ednote">
 				<h3>EPUB 3.4: 2027</h3>
 
-				<p>The work on EPUB 3.4 [[epub-34]] started in 2025. The goal was to add some new features 
-					to EPUB 3.3, and to clarify some existing features. The new features include the ability to
-					add standard annotating features to EPUB publications, and the ability to include 
-					"webtoons" in EPUB publications. The new features are expected to be published as
-					 a W3C Recommendation in 2027.</p>
+				<p>The work on EPUB 3.4 [[epub-34]] started in 2025. The goal was to add some new features to EPUB 3.3,
```

Comment:
```suggestion
				<p>The work on EPUB 3.4 [[epub-34]] started in 2025. The goal was to add new features to EPUB 3.3,
```
removed “some” if we feel a modifier is needed, can we be more specific? “community-requested” or “useful” or ?

Reply:
Note that this section is more of a placeholder and, at some point, must be made more precise. At this point, we have also removed things (well deprecated things) and not only added...

Reply:
Right, I'd leave this for now. The overview usually gets a final pass review to be more specific once we know what's definitely in and out.